lib/web_ui/lib/src/engine/util.dart

```diff
@@ -45,15 +45,27 @@ typedef Callbacker<T> = String? Function(Callback<T> callback);
 ///   return _futurize(_doSomethingAndCallback);
 /// }
 /// ```
+// Note: this function is not directly tested so that it remains private, instead an exact
+// copy of it has been inlined into the test at lib/ui/fixtures/ui_test.dart. if you change
+// this function, then you  must update the test.
```

Comment:
Isn't this function now public, and directly testable, instead of having to keep in sync two copies?

Reply:
Totally! I added tests.

Reply:
Cool!